node/test/go/dev-mode/test/lib/process-function.js

```diff
@@ -212,9 +222,11 @@ const retrieveReports = async (testConfig) => {
         continue;
       }
       if (message.startsWith('⚡ DEV-MODE: ReqRes###')) {
-        getCurrentInvocationData().reqRes.push(
-          JSON.parse(message.slice(message.lastIndexOf('###') + 3))
-        );
+        getCurrentInvocationData().reqRes.push(message.slice(message.lastIndexOf('###') + 3));
```

Comment:
How it is written to logs now? Is this base64 encoded payload?

Reply:
Yeah exactly 👍 when the proto data is written into the logs it gets automatically baseless encoded

Reply:
> it gets automatically base64 encoded

How exactly it happens automatically?

Reply:
[Right here](https://github.com/serverless/console/pull/237/files#diff-5db3564116d96759f718e99807a34ca6f143f0183607bc2a93b5d5f2c09c16fdR107). I found that when you convert a set of `[]byte` to `string` it was defaulting to base64 encoding. 

I ran a test on this on AWS with this extension enabled and passing a standard string into the API payload also if I log out the results in the integration tests the data is base64 encoded.

<img width="1282" alt="Screen Shot 2022-10-06 at 8 47 58 AM" src="https://user-images.githubusercontent.com/1753824/194330360-df5cb03c-db5f-4a0e-9d10-0e460e97f715.png">

Reply:
Yeah no problem it was good for me to double check 😎 

And you are right that string is suspiciously short because it is a fake proto payload I was using for testing with just some dummy text in it 😎